packages/ifpack2/src/Ifpack2_LocalSparseTriangularSolver_def.hpp

```diff
@@ -85,6 +85,7 @@ class LocalSparseTriangularSolver<MatrixType>::HtsImpl {
   }
 
   void setParameters (const Teuchos::ParameterList& pl) {
+    (void)pl;
```

Comment:
@mhoemmen, is this solution acceptable to you?  I don't like it, but I'm not sure what else to do without getting complicated with `#ifdef`s.  I don't think Trilinos has an accepted way of indicating unused parameters, does it?

Reply:
We've tried different things before.  Casting to `void` doesn't always work.  If it's really a problem, we could try adding a macro that does `__attribute__((unused))` (for GCC) or `[[maybe_unused]]` (C++17) if applicable.  As a last resort, I try complicated `#ifdef` stuff.  This matters in header files (both `_decl` and `_def`), since Sierra builds with warnings as errors.